ui/src/index.css

```diff
@@ -1,5 +1,10 @@
-body {
+* {
```

Comment:
What this is doing? Can't this be applied to a specific component?

Reply:
I added this to remove the presence of double scroll bars initially. + it was something I used to do for a long time as "good practice" 😬 . 

>Can't this be applied to a specific component?

Okay. I'll change

Reply:
Changed.